gov.pnnl.goss.gridappsd/src/gov/pnnl/goss/gridappsd/dto/CommunicationFault.java

```diff
@@ -0,0 +1,37 @@
+package gov.pnnl.goss.gridappsd.dto;
+
+import java.io.Serializable;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+
+public class CommunicationFault implements Serializable {
```

Comment:
Since we are differentiating between comm outage and faults. This class should be called CommunicationOutage and not CommunicationFault.

Reply:
Why do we need this class if we already have CommunicationFaultData Class?

Reply:
Move this to gov.pnnl.goss.gridappsd.dto.events folder.